src/PageLayout/PageHeader.js

```diff
@@ -14,47 +17,39 @@ class PageHeader extends Component {
             showPopUp: false,
             notificationClicked: false,
             notifications: [],
-            notifcationIndex: 0
+            notificationIndex: 0
         };
 
         this.notificationClicked = this.notificationClicked.bind(this);
         this.openPopUp = this.openPopUp.bind(this);
         this.closePopUp = this.closePopUp.bind(this);
     }
+
     componentDidMount() {
-        //TODO: query db for notifications
-        this.setState({
-            notifications: [
-                {
-                    type: 'recurring_pickup_request',
-                    content: '-L5QoXeC_UrL5tRRED3e'
-                },
-                {
-                    type: 'recurring_pickup_confirmed',
-                    content: '-XKSIDLeC_Uksd321e'
-                },
-                {
-                    type: 'recurring_pickup_rejected_dg',
-                    content: '-XKSIDLeC_Uksd321e'
-                },
-                {
-                    type: 'recurring_pickup_rejected_ra',
-                    content: '-XKSIDLeC_Uksd321e'
-                },
-                {
-                    type: 'recurring_pickup_unavailable',
-                    content: '-XKSIDLeC_Uksd321e'
-                }
-            ] 
-        });
+        let ref;
+        if (this.props.account.accountType === AccountType.DONATING_AGENCY_MEMBER) {
+            ref = db.ref(`donating_agencies/${this.props.account.agency}/notifications`);
+        } else {
+            ref = accountsRef.child(`${this.props.account.uid}/notifications`);
+        }   
+        ref.on('value', (snap) => this.setState({notifications: snap.val() ? snap.val() : []}));
+    }
+
```

Comment:
Maybe we could refactor the code here by saving `ref` as a `state` since these two blocks are almost identical. These two functions will only be triggered once for a lifecycle so we don't have to worry about the _state changes triggering re-rendering_ issue here.